scout/commands/delete/delete_command.py

```diff
@@ -8,6 +8,106 @@
 LOG = logging.getLogger(__name__)
 
 
+@click.command("variants", short_help="Delete variants for one or more cases")
+@click.option("-u", "--user", help="User running this command (email)", required=True)
+@click.option("-c", "--case-id", help="Case id")
+@click.option(
+    "-status",
+    type=click.Choice(["solved", "archived", "migrated", "active", "inactive", "prioritized"]),
+    multiple=True,
+    default=["solved", "archived", "migrated"],
+    help="Restrict to cases with specified status",
+)
+@click.option("-older-than", type=click.INT, default=0, help="Older than (months)")
+@click.option("-rank-threshold", type=click.INT, default=5, help="With rank threshold lower than")
+@click.option("-variants-threshold", type=click.INT, help="With more variants than")
+@click.option(
+    "--dry-run",
+    is_flag=True,
+    help="Perform a simulation without removing any variant",
+)
+@with_appcontext
+def variants(
+    user,
+    case_id,
+    status,
+    older_than,
+    rank_threshold,
+    variants_threshold,
+    dry_run,
+):
+    """Delete variants for one or more cases"""
+
+    user_obj = store.user(user)
+    if user_obj is None:
+        click.echo(f"Could not find a user with email '{user}' in database")
+        return
+
+    total_deleted = 0
+    items_name = "deleted variants"
+    if dry_run:
+        click.echo("--------------- DRY RUN COMMAND ---------------")
```

Comment:
```suggestion
        LOG.info("--------------- DRY RUN COMMAND ---------------")
```

Reply:
Only a suggestion, do as you prefer. It makes sense if you refactor this to own functions to use logging

Reply:
I am a bit hesitant here because I'm not sure how this is going to be written in the log. Specifically:
- This should be runned either on hasta or clinical-db, cli. Does the clinical-db write to a log file? I'm not sure.
- Since the command might take quite some time, what is some other command writes stuff in between on the log file? It would be a mess..

Reply:
I'll keep it like this for now. We can change it later